internal/pkg/template/input/validation_test.go

```diff
@@ -196,6 +196,17 @@ func TestValidationRules(t *testing.T) {
 			},
 			error: "",
 		},
+		{
+			description: "missing componentId for oauth kind",
+			input: Input{
+				Id:          "input.oauth",
+				Name:        "oauth",
+				Description: "oauth desc",
+				Type:        "object",
+				Kind:        "oauth",
+			},
+			error: "group 1, step 1, input \"input.oauth\": componentId is a required field",
```

Comment:
Tady jsou všechny ty inputy pro validaci nadefinovaný v poli a na konci se proženou `for` cyklem a zvalidují. Zkrátka se vezme `input`, zvaliduje se a zkontroluje že skončí s chybou která odpovídá `error`.